draft-ietf-masque-connect-ip.md

```diff
@@ -1392,9 +1392,12 @@ based on these markings. However, the use of HTTP can limit the possibilities
 for differentiated treatment of the tunneled IP packets on the path between the
 IP proxying endpoints.
 
-If tunneled packets are subject to congestion control by the outer connection,
-the tunneled packets need to be treated equally regardless of their DSCP
-markings to not disrupt the congestion controller. In this scenario, the IP
+When an HTTP connection is congestion-controlled, marking packets with
+different DSCP can lead to reordering between them, and that can in turn lead
```

Comment:
It not only reordering. Also getting different delay samples can break delay based cc.

Reply:
Do we need to go that much into detail?

Reply:
I think you simplification works fine here. As the relevant aspect cut out appear to covered in later sentences.

Reply:
I know this has been merged now but this comment of mine was addressed. reordering alone is not the problem.